modules/upgrade/pages/proxy-minor-uyuni.adoc

```diff
@@ -0,0 +1,37 @@
+[[proxy-package-uyuni]]
+= Proxy - Minor Upgrade
+
+Before you perform any proxy update, schedule a maintenance window.
+The clients registered to {productname} through the proxy will not be able to connect to {productname} while the update is in progress.
+For more information about maintenance windows, see xref:administration:maintenance-windows.adoc[].
+
+[NOTE]
+====
+Minor proxy upgrades do not include a version upgrade of the operating system.
+For example, while upgrading from 2021.06 to 2021.08 the operating system will stay at {opensuse} Leap 15.3.
```

Comment:
Do we really want to mention this? I'd just tell the user to check the release notes, to see if a major upgrade is needed.

Otherwise I doubt will remember to update this next time.

Reply:
ok, but we'd better check this always.